src/radiation/radiation_system.hpp

```diff
@@ -29,6 +29,8 @@
 #include "radiation/planck_integral.hpp"
 #include "util/valarray.hpp"
 
+#define LARGE 1.e100
+
```

Comment:
Can you get rid of this #define here, or move it into a done of the source term files?

Reply:
I moved it to the file for the dust model.